src/OpenLoco/src/Objects/RegionObject.h

```diff
@@ -25,7 +25,7 @@ namespace OpenLoco
         uint8_t pad_06[0x8 - 0x6];
         uint8_t var_08;
         uint8_t var_09[4];
-        uint8_t pad_0D[0x12 - 0xD];
+        uint8_t var_0D[5];
```

Comment:
This was changed for unrelated work, and should probably be reverted for this PR.

Reply:
```suggestion
        uint8_t pad_0D[0x12 - 0xD];
```